lib/appium/AppiumRunner.js

```diff
@@ -339,6 +340,7 @@ function getPackagePath (options) {
     case utilities.ANDROID:
         let packagePath = null;
         const maybePackagePaths = [
+            path.join(fullAppPath, 'platforms/android/app/build/outputs/apk/debug/app-debug.apk'),
```

Comment:
This should pass the path components separately (like the other two below) so that they use the correct path separator on all OSes

Reply:
> The path.join() method joins all given path segments together using the platform-specific separator as a delimiter, then normalizes the resulting path.

It should convert to the correct path separator for each platform.

Reply:
```suggestion
            path.join(fullAppPath, 'platforms', 'android', 'app', 'build', 'outputs', 'apk', 'debug', 'app-debug.apk'),
```